### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by ram-cherukuri**
Adding additional details for researchers

# Modulus Pull Request

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [x] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [ ] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

**Comment by ram-cherukuri** [comment]
@ktangsali See the example from Pytorch where they describe some key components - can we emulate the same for our readme? Need your help with that.

**Comment by ktangsali** [comment]
> @ktangsali See the example from Pytorch where they describe some key components - can we emulate the same for our readme? Need your help with that.

Thanks @ram-cherukuri - yes, I have been reviewing some great Readmes with @sharma-smriti. Have a few key insights. Will submit an update to this PR soon

**Review by ktangsali — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -65,13 +162,113 @@ python
...
+[NVIDIA Blogs](https://developer.nvidia.com/blog/category/simulation-modeling-design/)?
+Fill out this [proposal form](https://forms.gle/XsBdWp3ji67yZAUF7) and we will get back
+to you!
+
+## Why are they using Modulus
+
+Here are some of the key benefits of Modulus for SciML model development:
+
+<!-- markdownlint-disable -->
+<img src="docs/img/Value%20prop/benchmarking.svg" width="100"> | <img src="docs/img/Value%20prop/recipe.svg" width="100"> | <img src="docs/img/Value%20prop/performance.svg" width="100">
```
Comment (mnabian) [inline README.md:223]: These paths should be fixed. The icons don't show up in its current form.

**Review by mnabian — APPROVED** [review]
Please fix the paths to svg files before merging.

File: README.md (hunk 1 above)
Comment (ktangsali) [reply README.md:223]: Great catch! I forgot to update these. Will change.

**Pull request merged**